<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>278</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comL"/>
    </item>
    <item>
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>OPEN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>CLOSE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="clampButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>CLAMP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_1_lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_2_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_3_lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_4_lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_5_lineEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="servo_6_lineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>93</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_posButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SET POSITION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stand_upButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stand UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="S1_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="S2_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="S3_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="S4_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="S5_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="S6_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo 6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sitButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getBoxButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Box</string>
   </property>
  </widget>
  <widget class="QSlider" name="S6_verSlider">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="S5_verSlider">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="S4_verSlider">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="S3_verSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="S2_verSlider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="S1_verSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>93</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="horse_1_Button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Horse-1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="horse_2_Button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Horse-2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="put_getBoxButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PUT + GET Box</string>
   </property>
  </widget>
  <widget class="QPushButton" name="combSitStandButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SIT + Stand UP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
